mlflow/sagemaker/container/__init__.py

```diff
@@ -87,35 +99,52 @@ def _serve():
            "mlflow.sagemaker.container.scoring_server.wsgi:app").format(nworkers=cpu_count)
     bash_cmds.append(cmd)
     gunicorn = Popen(["/bin/bash", "-c", "; ".join(bash_cmds)])
-    signal.signal(signal.SIGTERM, lambda a, b: _sigterm_handler(nginx.pid, gunicorn.pid))
+    signal.signal(signal.SIGTERM, lambda a, b: _sigterm_handler(pids=[nginx.pid, gunicorn.pid]))
     # If either subprocess exits, so do we.
-    pids = set([nginx.pid, gunicorn.pid])
-    while True:
-        pid, _ = os.wait()
-        if pid in pids:
-            break
-    _sigterm_handler(nginx.pid, gunicorn.pid)
+    awaited_pids = _await_subprocess_exit_any(procs=[nginx, gunicorn])
+    _sigterm_handler(awaited_pids)
+
+
+def _serve_mleap(model):
+    mleap = Popen(["java", "-cp", "/opt/mlflow/mlflow/java/target/mlflow-java-"
+                   "{mlflow_version}-with-dependencies.jar".format(
```

Comment:
We could use `*` syntax in the Java `-cp` to avoid specifying version. It may be a good idea to keep the version in the file for clarity, but avoid requiring the mlflow version is the same for the built-in container.